packages/plugin-ext/src/plugin/window-state.ts

```diff
@@ -50,16 +50,20 @@ export class WindowStateExtImpl implements WindowStateExt {
     }
 
     openUri(uri: URI): Promise<boolean> {
+        if (!uri.scheme.trim().length) {
+            throw new Error('Invalid scheme - cannot be empty');
+        }
+
+        if (Schemes.http !== uri.scheme && Schemes.https !== uri.scheme) {
```

Comment:
T.b.h, I don't think it makes sense to check the scheme at all here: see the definition of `openExternal` in https://code.visualstudio.com/api/references/vscode-api#env. It's explicitly mentioned that an URI could be of the form `mailto://...`.

Reply:
Thank you for the review, Thomas! Good point, I removed the check.